### PR TITLE
add bastion_private_ip and master_lb_private_ip

### PR DESCRIPTION
**Pull request by clasohm**

### CODE REVIEW

File: activate-internal-lb.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        state: restarted
+
+- hosts: localhost
+  become: yes
+  tasks:
+    - name: Configure local client for API load balancer
+      lineinfile:
+        path: "~/.kube/config"
+        regexp: "    server: https://[^:]+:{{ hostvars['ocp-master-1']['openshift_master_api_port'] }}"
+        line: "    server: https://{{ hostvars['ocp-master-1']['master_lb_private_dns'] }}:{{ hostvars['ocp-master-1']['openshift_master_api_port'] }}"
```
Comment (hornjason) [inline activate-internal-lb.yml]: Should this be dynamic using groups?


        regexp: "    server: https://[^:]+:{{ hostvars[groups['masters'][0]]['openshift_master_api_port'] }}"

Comment (hornjason) [inline activate-internal-lb.yml]: +        line: "    server: https://{{ hostvars[groups['masters'][0]]['master_lb_private_dns'] }}:{{ hostvars[groups['masters'][0]]['openshift_master_api_port'] }}"

File: roles/azure_infra/templates/hosts.j2 [hunk 2]
```diff
@@ -41,8 +41,13 @@ openshift_master_console_port=443
 openshift_hosted_router_replicas=3
 openshift_hosted_registry_replicas=3
 openshift_master_cluster_method=native
+{% if master_lb_private_ip is not defined %}
 openshift_master_cluster_hostname={{ rg }}.{{ location }}.cloudapp.azure.com
 openshift_master_cluster_public_hostname={{ rg }}.{{ location }}.cloudapp.azure.com
+{% else %}
+openshift_master_cluster_hostname=ocp-master-1
```
Comment (hornjason) [inline roles/azure_infra/templates/hosts.j2]: openshift_master_cluster_hostname={{ groups['masters'][0] }}

**Comment by hornjason** [comment]
added some comments instead of hardcoding master1 use variables.

File: activate-internal-lb.yml (hunk 1 above)
Comment (clasohm) [reply activate-internal-lb.yml]: Changed in commit 7a94f5c.

Comment (clasohm) [reply activate-internal-lb.yml]: Changed in commit 7a94f5c.

File: roles/azure_infra/templates/hosts.j2 (hunk 2 above)
Comment (clasohm) [reply roles/azure_infra/templates/hosts.j2]: Changed in commit 7a94f5c.

**Pull request merged**